### PR TITLE
SAL operators

### LINKED ISSUE #4958: Add `LIKE` SQL operator documentation to SQL reference.
`LIKE` is so commonly used in SQL, we should probably document it.

##### Relevant URLs
- https://www.postgresql.org/docs/current/functions-matching.html#FUNCTIONS-LIKE

### PR DESCRIPTION
**Pull request by sanderson**
Closes #4958 

Adds detailed descriptions and examples of SQL operators. This initial draft only includes changes to the Dedicated docs. Once reviewed, I'll port everything over to Serverless and Clustered.

- [x] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/cloud-dedicated/reference/sql/operators/bitwise.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+| ------------------: |
+|                   1 |
+
+{{% /flex-content %}}
+{{< /flex >}}
+
+## \| {#bitwise-or .monospace}
+
+The `|` (bitwise OR or inclusive OR) operator compares each bit of the left
+operand to the corresponding bit of the second operand.
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/reference/sql/operators/bitwise.md:58]:
```suggestion
operand to the corresponding bit of the right operand.
```

File: content/influxdb/cloud-dedicated/reference/sql/operators/logical.md [hunk 2]
```diff
@@ -0,0 +1,463 @@
...
+
+{{% /influxdb/custom-timestamps %}}
+{{% /expand %}}
+
+{{% expand "`NOT EXISTS`" %}}
+
+```sql
+SELECT *
+FROM
+  home home_actions
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/reference/sql/operators/logical.md:367]: This doesn't require a comma?

File: content/influxdb/cloud-dedicated/reference/sql/operators/other.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+SQL supports miscellaneous operators that perform various operations.
+
+| Operator | Meaning             |                                             |
+| :------: | :------------------ | :------------------------------------------ |
+|  `\|\|`  | Concatenate strings | [{{< icon "link" >}}](#concatenate-strings) |
+
+## || {#concatenate-strings}
+
+The `||` operator concatenates two strings operands into a single string.
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/reference/sql/operators/other.md:25]:
```suggestion
The `||` operator concatenates two string operands into a single string.
```

**Review by jstirnaman — APPROVED** [review]
A few minor fixes(?). Otherwise, looks good - thanks!

File: content/influxdb/cloud-dedicated/reference/sql/operators/logical.md (hunk 2 above)
Comment (sanderson) [reply content/influxdb/cloud-dedicated/reference/sql/operators/logical.md:367]: No, it doesn't. This is the syntax for a correlated subquery: https://arrow.apache.org/datafusion/user-guide/sql/subqueries.html#exists

**Pull request merged**